File: Algos/groupAnagram.py
def groupAnagrams(strs):
    solution = []
    for i in range(0,len(strs)-1):
        solution.append([strs[i]])
        temp = strs[i] = ''.join(sorted(strs[i]))
        for j in range(i+1, len(strs)-1):
            temp2 = ''.join(sorted(strs[j]))
            if temp == temp2:
                solution[i].append(strs[j])
                del strs[j]
    return solution

strings = ["eat","tea","tan","ate","nat","bat"]

print(groupAnagrams(strings))

File: Algos/GroupAnagram.js
var strArr = ["eat","tea","tan","ate","nat","bat"]

var groupAnagrams = function (strs) {
    var sortedArr = [];
    for (var i = 0; i < strs.length; i++) {
        sortedArr[i]= [strs[i]];
        for (var j = i + 1; j < strs.length; j++)
            if (strs[i].sort((a, b) => a - b) == strs[j].sort((a, b)=> a - b)){
                sortedArr[i].push(strs[j])
                strs.splice(j,1)
                j--
            }
    }return sortedArr;
};

console.log(groupAnagrams(strArr))